runtime/karura/src/lib.rs

```diff
@@ -2188,7 +2188,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
-	(TransactionPaymentMigration, XcmInterfaceMigrationV1),
+	XcmInterfaceMigrationV1,
```

Comment:
XcmInterfaceMigrationV1 not applied?

Reply:
Not yet

Reply:
https://github.com/AcalaNetwork/Acala/pull/1893